README.md

```diff
@@ -72,6 +72,18 @@ Set the latest version:
 spin verman set latest
 ```
 
+Create/update a pointer to a custom version:
+
+```sh
+spin verman set custom --file path/to/spin/binary
```

Comment:
Similar to @karthik2804's point, it would be great to have something along the lines of:
```
spin verman set <user-chooses-name> /path
```
`verman` would know that it is a custom entry due to the path being provided